test/runtime/Policies.go

```diff
@@ -1302,3 +1303,101 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(res.Output().String()).Should(ContainSubstring(allowedVerdict), "Policy trace did not contain %s", allowedVerdict)
 	})
 })
+
+var _ = Describe("RuntimeValidatedPolicyValidationTests", func() {
```

Comment:
Could we change this to a new file called `runtime/Examples` I think that ti's a better place

Reply:
Done.